docs/community-ee/community-ee-release-process.md

```diff
@@ -0,0 +1,200 @@
+# Releasing the community-ee-* execution environments
+
+## Release Cadence
+
+The community EEs are to be built and published on the next day and/or the day after ansible-core is released.
+
+## EE tag versioning
+
+Versioning would be the core tag + a patch number, e.g
+
+    - EE with core `2.16.2` comes out -> `community-ee:2.16.2-1`
+    - EE with core `2.16.3` comes out -> `community-ee:2.16.3-1`
+
+
+## Dependencies
+
+-   Podman/Docker
+
+##  Credentials
+
+- Access to  https://github.com/ansible-community/images repo
+- Access to ghcr.io
+
+
+## Build steps for Community-ee-base
+
+1. Go to the working directory.
+
+`cd images/execution-environments/community-ee-base`
+
+2. Create a new release branch
+
+`git checkout -b base-<ansible-core-version-base-ee-version>`
+
+3. Edit the `execution-environment.yml` file to the right `collection version` and `ansible-core version`. Get the correct [ansible-core](https://pypi.org/project/ansible-core/) version and [ansible-<version>.deps](https://github.com/ansible-community/ansible-build-data/blob/main/9/ansible-9.0.1.deps) and edit accordinly.
+
+`vim execution-environments/community-ee-base/execution-environment.yml`
+
+```
+version: <base-community-ee with the specific ansible core>
+images:
+  base_image:
+    name: quay.io/fedora/fedora:latest
+dependencies:
+  ansible_core:
+    package_pip: ansible-core==<ansible-core-version>
+  ansible_runner:
+    package_pip: ansible-runner
+  system:
+    - openssh-clients
+    - sshpass
+  galaxy:
+    collections:
+    - name: ansible.posix
+      version: <correct-version>
+    - name: ansible.utils
+      version: <correct-version>
+    - name: ansible.windows
+      version: <correct-version>
+
+```
```

Comment:
I would remove this, since it's basically a copy of the EE definition in the repo, and if we forget to update this copy when we edit the main source of truth, it will quickly get confusing.
```suggestion
```